react/components/SearchBar/components/ResultsList.js

```diff
@@ -1,116 +1,122 @@
-import React, { Component, Fragment } from 'react'
+import React, { Fragment } from 'react'
+import classnames from 'classnames'
 import PropTypes from 'prop-types'
 import { graphql } from 'react-apollo'
 import { Spinner } from 'vtex.styleguide'
-import { Link } from 'vtex.render-runtime'
+import { Link, useRuntime } from 'vtex.render-runtime'
 
 import autocomplete from '../queries/autocomplete.gql'
 
 import styles from '../styles.css'
 
-const listClassNames =
-  `${styles.resultsList} z-max absolute w-100 mt1 mt2-m pb4 bg-white f5 left-0`
-const listItemClassNames =
-  `${styles.resultsItem} flex justify-start f5 pa4 pl6 outline-0`
+/** List of search results to be displayed*/
+const ResultsList = ({
+  data,
+  emptyPlaceholder,
+  inputValue,
+  closeMenu,
+  onClearInput,
+}) => {
+  const items = data.autocomplete ? data.autocomplete.itemsReturned : []
```

Comment:
Do you think that using Ramda's `pathOr` it would look better? 
```suggestion
  const items = pathOr([], ['autocomplete', 'itemsReturned'], data)
```